Documentation/Books/Manual/Security/Auditing/AuditConfiguration.md

```diff
@@ -27,4 +27,16 @@ Hostname
 `--audit.hostname name`
 
 The name of the server used in audit log messages. By default the
-system hostname is used.
\ No newline at end of file
+system hostname is used.
+
+Verbosity
+---------
+
+`--log.level topic=level`
+
+By default, the server will log all audit events. Some low-priority events, such
+as statistics operations, are logged with the `debug` log level. To keep such
+events from cluttering the log, set the appropriate topic to `info`. All other
+messages will be logged at the `info` level. Audit topics include
+`audit-authentication`, `audit-collection`, `audit-database`, `audit-documents`,
+`audit-service`, and `audit-views`. 
```

Comment:
These topics don't seem to match what is written here: https://docs.arangodb.com/devel/HTTP/AdministrationAndMonitoring/index.html#modify-and-return-the-current-server-log-level

- `audit-documents` vs. `audit-documentation`
- `audit-views` vs. `audit-view`

I don't know what is correct, can you please adjust it in this file or the [other one](https://github.com/arangodb/arangodb/edit/feature/improved-audit-logging/Documentation/DocuBlocks/Rest/Administration/get_admin_modules_flush.md)?